setup.py

```diff
@@ -3,9 +3,9 @@
     name='consensus',
     version='1.0',
     packages=find_packages(),
-    install_requires=['molgenis-py-client>=2.1.0', 'progressbar2==3.39.3', 'termcolor==1.1.0', 'yaspin==0.14.3', 'pandas'],
+    install_requires=['progressbar2==3.39.3', 'termcolor==1.1.0', 'yaspin==0.14.3', 'pandas'],
```

Comment:
Is there an explicit requirement for the panda version? (f.e. does it work with a `0.x` version: https://pypi.org/project/pandas/#history)

Reply:
I'm not sure. I don't know a thing about pandas. I dislike them as much as I dislike camels. They do however make my code faster. I'll put the version number I use at the moment in there. I'm sure it works with that version.